src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -120,6 +120,11 @@ public void setPerson(Person target, Person editedPerson) {
         addressBook.setPerson(target, editedPerson);
     }
 
+    @Override
+    public Boolean tagExists(Person target, Set<Tag> tags) {
+        return target.tagExists(tags);
+    }
+
```

Comment:
Should add java docs to this

Reply:
I think unnecessary to add java docs to overridden methods that do not significantly modify the overridden method